src/Tracking.lua

```diff
@@ -51,11 +51,11 @@ local function OnCombatEvent(setKey, _, result, _, abilityName, _, _, _, _, _, _
 
     if result == ACTION_RESULT_ABILITY_ON_COOLDOWN then
         Cool:Trace(1, "<<1>> (<<2>>) on Cooldown", abilityName, abilityId)
-    elseif result == set.result then
+    elseif result == set.result and not set.onCooldown then
```

Comment:
Good addition. Makes me even more sad that `ACTION_RESULT_ABILITY_ON_COOLDOWN` is no longer in the game.